### ISSUE TITLE
There is no documentation about how to respond a custom http status code

### ISSUE THREAD

**Issue by sdelamo** [micronaut-projects/micronaut-core#11220]
support for custom status codes was introduced here: https://github.com/micronaut-projects/micronaut-core/pull/8147

**Comment by Nitishrajj** [comment]
Hi, If this issue is not yet resolved, I would like to take it up.

**Comment by yawkat** [comment]
@Nitishrajj sure. Though be aware this is just about the docs, it will not be a code change

**Comment by Nitishrajj** [comment]
@yawkat Yes I am well aware of that. As a newbie to  opensource community, I would like to contribute with something small and eventually go big.

**Comment by yawkat** [comment]
Reasonable attitude. Feel free to work on this.

**Comment by Nitishrajj** [comment]
Glad to work, thank you.

**Comment by Nitishrajj** [comment]
https://github.com/micronaut-projects/micronaut-core/pull/11445

Please refer this

**Comment by sdelamo** [comment]
Close via #11445
